Fatras/include/ActsFatras/EventData/Hit.hpp

```diff
@@ -45,13 +45,8 @@ class Hit {
   /// position on the given surface.
   Hit(Acts::GeometryIdentifier geometryId, Barcode particleId,
       const Vector4& pos4, const Vector4& before4, const Vector4& after4,
-      int32_t index_ = -1)
-      : m_geometryId(geometryId),
-        m_particleId(particleId),
-        m_index(index_),
-        m_pos4(pos4),
-        m_before4(before4),
-        m_after4(after4) {}
+      int32_t index_ = -1);
+
```

Comment:
Does this need to be moved into a cpp file or could we leave the hit implementation as-is?